lib/tasks/users_lookup.rake

```diff
@@ -1,8 +1,10 @@
 namespace :users do
   desc 'Look up a user by email address'
   task lookup_by_email: :environment do |_task, args|
-    print 'Enter the email address to look up: '
-    email = gets.strip
+    require 'io/console'
+    print 'Enter the email address to look up (input will be hidden): '
+    email = STDIN.noecho(&:gets).strip
+    puts "\n"
```

Comment:
if `'io/console'` is required, we have [`getpass`](https://www.rubydoc.info/stdlib/io/IO:getpass) which does all this, and even has the little :key: on my mac
```suggestion
    require 'io/console'
    email = STDIN.getpass('Enter the email address to look up (input will be hidden): ')
```

example:
```
STDIN.getpass("email: ")
email: 
=> "aaaaaa"
```

There are gems and tools that can also give feedback as you type, which I really enjoy but that adds a lot of dependencies...

```
require 'tty-prompt'
TTY::Prompt.new.mask('email:', mask: '*')
email: **************
=> "aaaaaa@aaa.com"
```

Reply:
I think `getpass` would be sufficient here?